templates/operations.ts.hbs

```diff
@@ -186,8 +187,11 @@
           method: "{{loud method}}"
         }
       );
+      if(returnTypeParam && returnTypeParam === returnType.REQUEST) {
```

Comment:
```suggestion
      if(returnTypeParam === returnType.REQUEST) {
```
Checking for truthiness is unnecessary if you're just doing an equality check. 😉